azurerm/internal/services/datafactory/data_factory_dataset_azure_blob_resource.go

```diff
@@ -72,6 +71,18 @@ func resourceDataFactoryDatasetAzureBlob() *pluginsdk.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"is_path_dynamic": {
```

Comment:
could we make thse consistant with the rest of the provider?
```suggestion
			"dynamic_paths_enabled": {
```